# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/xjdic/hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./xjdxgen.c_old	2003-02-04 17:32:52.000000000 +0100
+++ ./xjdxgen.c	2003-02-04 17:32:23.000000000 +0100
@@ -35,7 +35,7 @@
 #define TOKENLIM 40
 
 unsigned char *db;
-unsigned char ENVname[50];
+unsigned char ENVname[150];
 unsigned char *dicenv;
 struct stat *buf;
 unsigned long dbyte;
